server/routes/user.js

```diff
@@ -53,7 +53,7 @@ userRouter.get(`/stripe/callback/`, async (req, res) => {
     const balance = await stripe.balance.retrieve({
       stripeAccount: response.stripe_user_id,
     });
-    user.update({
+    await user.update({
       stripe: response.stripe_user_id,
```

Comment:
was getting an error on a user.update somewhere in the project when it builds, seems like this was the fix?